frontend/src/pages/modelCatalog/screens/ModelCatalog.tsx

```diff
@@ -15,16 +15,35 @@ const ModelCatalog: React.FC = conditionalArea(
 )(() => {
   const { modelCatalogSources } = React.useContext(ModelCatalogContext);
   const renderStateProps = {
```

Comment:
I think we can take this whole `renderStateProps` object and move its stuff as directly passing props to ApplicationsPage below, like we have for ModelDetailsPage. It was lifted out in other pages because it was conditional / it would vary, but there's no reason for that here.